src/Microsoft.Health.ControlPlane.Core/Features/Rbac/IdentityProvider.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.ControlPlane.Core.Features.Rbac
+{
+    public class IdentityProvider
+    {
+        [JsonConstructor]
+        protected IdentityProvider()
+        {
+        }
+
+        public IdentityProvider(string name, string authority, IReadOnlyList<string> audience, string version)
+        {
+            Name = name;
+            Authority = authority;
+            Audience = audience;
+            Version = version;
+        }
+
+        [JsonProperty("Name")]
```

Comment:
The other documents uses JSON format with camel case so we should follow the same.